Adyen/Integration/01_ManageAdyenSettings.md

```diff
@@ -0,0 +1,282 @@
+
+2. Copy the username.
+
+3. Change to your Actindo instance. Enter the username in the *Payment user* field.
+ 
+     ![Payment user](../../Assets/Screenshots/Adyen/Integration/AdyenWebServiceUser.png "[Payment user]")
+
+4. Return to your Adyen&trade; account.
+
+5. Enter a description for the web service user in the *Description* field.
+
+    ![Create API credential](../../Assets/Screenshots/Adyen/Integration/AdyenCreateAPIcredential.png "[Create API credential]")
+
+5. Click the [Create credential] button.   
+    The *Configure API credential* view is displayed. The API key tab is displayed by default.
+
+6. Generate the *Web service* API key and enter it in Actindo. To do this, follow these steps:
+
+    - Click the *API key* tab in the *Server settings > Authentication* section. 
+
+    - Click the [Generate API key] button.    
+      The API key is automatically generated.
+
+        ![Generate checkout API key](../../Assets/Screenshots/Adyen/Integration/AdyenAuthentication.png "[Generate checkout API key]")
+    
+    - Click the [Copy] button.
+
+    - Change to your Actindo instance and insert the API key in the *Checkout API key* field. 
+
+        ![Checkout API key](../../Assets/Screenshots/Adyen/Integration/AdyenCheckoutAPIkey.png "[Checkout API key]")
+
+7. Return to your Adyen&trade; account and generate the *Web service* API password and enter it in Actindo. To do this, follow these steps:
+
+    - Click the *Basic auth* tab and click the [Generate password] button.
+      The password is automatically created.
+
+        ![Web service password](../../Assets/Screenshots/Adyen/Integration/AdyenWebServicePassword.png "[Web service password]")
+    
+    - Click the [Copy] button. 
+
+    - Change to your Actindo instance and insert the password in the *Payment password* field.
+
+       ![Payment password](../../Assets/Screenshots/Adyen/Integration/AdyenPaymentPassword.png "[Payment password]")
+
+7. Return to Adyen&trade; and click the [Save changes] button in the bottom right corner.   
+    The credentials for the *Web service* API are applied.
+
+8. If necessary, configure your wallet payment methods in the *Wallet payment methods* section. Follow the Adyen&trade; documentation. 
+
+9. If necessary, assign roles to change the permissions of the API credentials in the *Permissions > Roles* section. For dteailed information, see [API permissions](https://docs.adyen.com/development-resources/api-credentials/#api-permissions "[https://docs.adyen.com/development-resources/api-credentials/#api-permissions]") in the Adyen&trade; documentation.
+
+
+
+## Create report service API credentials
+
+Create the report service API credentials to send messages from the *Payments* module to the Adyen&trade; backend. You can use this report service to ensure that all messages have been successfully transferred from Adyen&trade; to the *Payments* module.
```

Comment:
"Create the report service API credentials to send messages from the Payments module to the Adyen™ backend." --> Dieser Satz kommt unter "Crate web servive API credentials" vor. Was ist der Unterschied zwischen beiden Funktionen? Vielleicht das Thema "report" hier im Vordergrund stellen/zuerst erwähnen?